ci/travis_jobs/docker_setup.sh

```diff
@@ -3,12 +3,13 @@ echo 'doing docker build'
 # Note: adding --build-arg <arg-name> without any value tells docker to
 #  use value from local environment (export DO_GIT_CLONE)
 
-#${TRAVIS_BUILD_DIR}/ci/travis_jobs/get_data_volumes.py
+${TRAVIS_BUILD_DIR}/ci/travis_jobs/get_data_volumes.py
```

Comment:
This shouldn't be done in this script.

Reply:
Ok.  It was there when I cloned the repo.  I wasn't sure about it